### PR TITLE
feat: add test for limit

### PR DESCRIPTION
**Pull request by VebjornG**
[Ticket](https://cognitedata.atlassian.net/browse/AH-3372?atlOrigin=eyJpIjoiZjJjOTczZWYyNzYyNDY4YjhkZDE0NDk4OThhM2Y1YTMiLCJwIjoiaiJ9)

### CODE REVIEW

**Review by tuanng-cognite — APPROVED** [review]
Nice! I thought it did not support the limit

File: packages/beta/src/__tests__/api/alertsApi.int.spec.ts [hunk 1]
```diff
@@ -189,15 +189,21 @@ describe('alerts api', () => {
...
+      },
+    });
+    expect(response.items.length).toBeGreaterThan(0);
+  });
+
+  test('test limit', async () => {
+    const response = await client.alerts.list({
+      limit: 1,
+    });
+    expect(response.items.length).toBe(1);
```
Comment (saagar-cognite) [inline packages/beta/src/__tests__/api/alertsApi.int.spec.ts]: question(blocking): How many alerts are already there ?

Comment (saagar-cognite) [reply packages/beta/src/__tests__/api/alertsApi.int.spec.ts]: I think we should create 2 alerts and then check for limit: 1 and see if it returns just 1.

Comment (VebjornG) [reply packages/beta/src/__tests__/api/alertsApi.int.spec.ts]: I'm creating 10 alerts in my latest commit @saagar-cognite

**Review by saagar-cognite — APPROVED** [review]
LGTM!
Though sorting is not tested as a feature but it should be safe to assume that its implemented correctly in backend.

**Comment by saagar-cognite** [comment]
Just needs the fix of externalIds I guess.

**Pull request merged**